src/numbers.js

```diff
@@ -1,45 +1,46 @@
-function add (a, b) {
-  // your code here
-}
+const add = (a, b) => {
+  return a + b;
+};
 
-function subtract (a, b) {
-  // your code here
+function subtract(a, b) {
+  return a - b;
 }
 
-function multiply (a, b) {
-  // your code here
+function multiply(a, b) {
+  return a * b;
 }
 
-function divide (a, b) {
-  // your code here
+function divide(a, b) {
+  return a / b;
 }
 
-function power (a, b) {
-  // your code here
+function power(a, b) {
+  return a ** b;
 }
 
-function round (a) {
-  // your code here
+function round(a) {
+  return Math.round(a);
 }
 
-function roundUp (a) {
-  // your code here
+function roundUp(a) {
+  return Math.ceil(a);
 }
 
-function roundDown (a) {
-  // your code here
+function roundDown(a) {
+  return Math.floor(a);
 }
 
-function absolute (a) {
-  // your code here
+function absolute(a) {
+  return Math.abs(a);
 }
 
-function quotient (a, b) {
-  // your code here
-}
+const quotient = (a, b) => {
+  // eslint-disable-next-line radix
+  return parseInt(a / b);
```

Comment:
Avoid disabling eslint rules whenever possible. In this case, it just wants you to add in the optional radix parameter. As you're working in base 10, use 10 as this parameter. i.e.
```
parseInt(a / b, 10);
```